node/grpc/server_test.go

```diff
@@ -83,6 +83,7 @@ func makeConfig(t *testing.T) *node.Config {
 		DbPath:                    t.TempDir(),
 		ID:                        opID,
 		NumBatchValidators:        runtime.GOMAXPROCS(0),
+		EnableV2:                  true,
```

Comment:
This removes the test coverage for the existing v1, we need to keep testing it

Reply:
Sounds good. Defaulted it to false and overwrote it to true in `server_v2_test`